ci-operator/step-registry/vsphere/provision/usertags/vsphere-provision-usertags-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: vsphere-provision-usertags
+  from: upi-installer
```

Comment:
If the step does not rely on installer, suggest to use a specific tag name of upi-installer.

Reply:
specific tag name? do you have an example?

Reply:
```
ref:
  as: cucushift-installer-check-vsphere-customized-resource
  from_image:
   namespace: ocp
   name: "4.14"
   tag: upi-installer
```